### PR TITLE
operbox描述

### PR DESCRIPTION
**Pull request by GD-GK**
英文文档有些地方可能描述不对

### CODE REVIEW

File: apps/doc/en-us/3.2-CALLBACK_SCHEMA.md [hunk 1]
```diff
@@ -542,5 +544,48 @@ Todo
...
+        },
+    ]
+    "own_opers": [
+        {
+            "id": "char_002_amiya", // operator id
+            "name": "阿米娅", // operator name
+            "own": true, // did you have this operator
+            "elite": 2, // operator elite 0,1,2
+            "level": 50, // operator level
+            "potential": 5 // operator potentital 0-5
```
Comment (ABA2396) [inline apps/doc/en-us/3.2-CALLBACK_SCHEMA.md:577]: "id": "char_002_amiya", // Operator ID
"name": "阿米娅", // Operator name
"own": true, // Whether owned
"elite": 2, // Elite level 0, 1, 2
"level": 50, // Operator level
"potential": 5 // Operator potential 0-5

File: apps/doc/en-us/3.2-CALLBACK_SCHEMA.md [hunk 2]
```diff
@@ -542,5 +544,48 @@ Todo
     // Only ArkPlanner and Lolicon formats are supported. More websites will be supported in future.
     ```
 
+- `OperBoxInfo`<br>
+    Recognition result of operator box
+
+    ```jsonc
+    // Example of corresponding details field
+    "done": bool,       // Has the recognition been completed? False indicates that the recognition is still in progress (data during the process)
```
Comment (ABA2396) [inline apps/doc/en-us/3.2-CALLBACK_SCHEMA.md:552]: // Whether the recognition has been completed, false means it is still in progress (data during the process)

File: apps/doc/en-us/3.1-INTEGRATION.md [hunk 3]
```diff
@@ -243,6 +243,16 @@ For more details about auto-copilot JSON, please refer to [Copilot Schema](3.7-S
 }
 ```
 
+- `OperBox`
```
Comment (ABA2396) [inline apps/doc/en-us/3.1-INTEGRATION.md:246]:
```suggestion
- `OperBox`<br>
```

File: apps/doc/ja-jp/3.1-統合ドキュメント.md [hunk 4]
```diff
@@ -210,6 +210,16 @@ TaskId ASSTAPI AsstAppendTask(AsstHandle handle, const char* type, const char* p
 }
 ```
 
+- `OperBox`
```
Comment (ABA2396) [inline apps/doc/ja-jp/3.1-統合ドキュメント.md:213]:
```suggestion
- `OperBox`<br>
```

File: apps/doc/zh-tw/3.1-集成文件.md [hunk 5]
```diff
@@ -210,6 +210,16 @@ TaskId ASSTAPI AsstAppendTask(AsstHandle handle, const char* type, const char* p
...
+- `OperBox`
+    Operator box recognition
+
+```jsonc
+// Corresponding task parameters
+{
+    "enable": bool          // Whether to enable this task, optional, by default true
+}
+```
+
```
Comment (ABA2396) [inline apps/doc/zh-tw/3.1-集成文件.md:222]:
```suggestion
- `OperBox`<br>
    幹員box識別

```jsonc
// 對應的任務參數
{
    "enable": bool          // 是否啟用本任務，可選，默認為 true
}
```

```

File: apps/doc/zh-tw/3.2-回呼訊息協定.md [hunk 6]
```diff
@@ -120,6 +120,8 @@ Todo
     自動公招
 - `RecruitCalc`<br>
     公招辨識
+- `OperBox`<br>
+    Operator Box Recognition
```
Comment (ABA2396) [inline apps/doc/zh-tw/3.2-回呼訊息協定.md:124]:
```suggestion
    幹員box識別
```

File: apps/doc/zh-tw/3.2-回呼訊息協定.md [hunk 7]
```diff
@@ -511,5 +513,48 @@ Todo
     // 目前只支援 ArkPlanner 和 Lolicon 的格式，以後可能會相容更多網站
     ```
 
+- `OperBoxInfo`<br>
+    Recognition result of operator box
```
Comment (ABA2396) [inline apps/doc/zh-tw/3.2-回呼訊息協定.md:517]:
```suggestion
    幹員識別結果
```

File: apps/doc/zh-tw/3.2-回呼訊息協定.md [hunk 8]
```diff
@@ -511,5 +513,48 @@ Todo
...
+
+    ```jsonc
+    // Example of corresponding details field
+    "done": bool,       // Whether the recognition has been completed, false means it is still in progress (data during the process)
```
Comment (ABA2396) [inline apps/doc/zh-tw/3.2-回呼訊息協定.md:521]:
```suggestion
    // 對應的 details 字段舉例
    "done": bool,       // 是否已經識別完了，為 false 表示仍在識別中（過程中的數據）
```

File: apps/doc/zh-tw/3.2-回呼訊息協定.md [hunk 9]
```diff
@@ -511,5 +513,48 @@ Todo
...
+        },
+    ]
+    "own_opers": [
+        {
+            "id": "char_002_amiya", // Operator ID
+            "name": "阿米娅", // Operator name
+            "own": true, // Whether owned
+            "elite": 2, // Elite level 0, 1, 2
+            "level": 50, // Operator level
+            "potential": 5 // Operator potential 0-5
```
Comment (ABA2396) [inline apps/doc/zh-tw/3.2-回呼訊息協定.md:546]:
```suggestion
            "id": "char_002_amiya", // 幹員id
            "name": "阿米娅", // 幹員名稱
            "own": true, // 是否擁有
            "elite": 2, // 精英度 0, 1, 2
            "level": 50, // 幹員等級
            "potential": 5 // 幹員潛能 0-5
```

**Comment by MistEO** [comment]
英文翻译用 deepl 吧，百度翻的实在不行

**Pull request merged**